apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -84,84 +68,66 @@ class MultipleChoiceOverviewTable extends Component {
     });
   };
 
-  getColumns = (sortable) => {
-    let dataColumns = [
-      {
-        property: 'question',
-        header: {
-          label: commonMsg.question(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-          }},
-          transforms: [sortable],
-        },
-        cell: {
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-          }}
-        }
+  answerOptions = (index) => (
+    {
+      header: {
+        label: alphabetMapper[index],
+        props: {
+          style: {
+          ...tableLayoutStyles.headerCell,
+        }},
       },
-      {
-        property: 'percentAnsweredOptionOne',
-        header: {
-          label: commonMsg.answerOptionA(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsOneFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
-      },
-      {
-        property: 'percentAnsweredOptionTwo',
-        header: {
-          label: commonMsg.answerOptionB(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsTwoFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+      cell: {
+        format: answerOptionsFormatter,
+        props: {
+          style: {
+          ...tableLayoutStyles.cell,
+        }}
+      }
+    }
+  );
+
+  questionColumn = (sortable) => (
```

Comment:
naming: something like `getQuestionColumn` or `createQuestionColumn` could imply you're actually making one.

Reply:
Fixed!